modules/jdbc/src/main/java/org/testcontainers/jdbc/ContainerDatabaseDriver.java

```diff
@@ -80,51 +69,34 @@ public synchronized Connection connect(String url, final Properties info) throws
         if (!acceptsURL(url)) {
             return null;
         }
+        
+        ConnectionUrl cUrl = new ConnectionUrl(url);
```

Comment:
Please could you call this parameter `connectionUrl`? Maybe it's just me, but every time I see `cUrl` I think of [this](https://curl.haxx.se/). In general we're quite OK with medium-verbosity names for things in this project 😄

Reply:
Changed now 😄